src/ontology/mondo-edit.obo

```diff
@@ -280052,11 +280052,11 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 id: MONDO:0014017
 name: intellectual developmental disorder with autism and macrocephaly
 subset: predisposition
-synonym: "autism, susceptibility to, 18" EXACT CLINGEN_PREFERRED [MONDO:Lexical, OMIM:615032]
```

Comment:
@nicolevasilevsky Is it ok to remove the "ClinGen_Preferred" from this term? 
You might have already checked that ClinGen is not using this name, but I want to make sure we are not removing this tag if they are still using this name.

Reply:
I'm not sure how to check if ClinGen is using this term. These tags were initially added with a list that they gave us but that was quite a while ago and I would think it is no longer up to date.